Signal/Images.xcassets/ic_speaker_bluetooth_inactive_audio_mode.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "ic_speaker_bluetooth_inactive_audio_mode.png",
+      "scale" : "1x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "2x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "3x"
+    }
+  ],
+  "info" : {
+    "version" : 1,
+    "author" : "xcode"
```

Comment:
> Where'd you get this asset?

I made it in an attempt to match our existing style, while matching platform conventions. In particular, I didn't want the speaker portion of the button to be positioned differently in the speakerphone vs. bluetooth.

>  Also, don't forget to update ...
Yup, I've got them in Signal-iOS-Design locally, waiting until this is merged.